frontend/src/components/FavoriteButton.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { Button } from '@patternfly/react-core';
+import { StarIcon } from '@patternfly/react-icons';
+import classNames from 'classnames';
+
+import './FavoriteButton.scss';
+
+type FavoriteButtonProps = {
+  isFavorite: boolean;
+  onClick: () => void;
+};
+
+const FavoriteButton: React.FC<FavoriteButtonProps> = ({ isFavorite, onClick }) => {
+  return (
+    <Button variant="plain" style={{ padding: 0 }} onClick={onClick}>
```

Comment:
Can we move this into the style file as well? I think once we follow BEM it will make more sense to you